<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logging_dialog</class>
 <widget class="QDialog" name="logging_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#logging_dialog{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 5, 68, 255), stop:1 rgba(10, 55, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="welcome_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 28pt &quot;Arial&quot;;
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>WELCOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="signin_label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 16pt &quot;Arial&quot;;
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>SIGN IN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>500</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;color: rgb(85, 255, 255);
background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>LOG IN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;color: rgb(85, 255, 255);
background-color: rgba(0,0,0,0);
border: none;
border-bottom: 4px solid rgba(46,82,101,200)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwd_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;color: rgb(85, 255, 255);
background-color: rgba(0,0,0,0);
border: none;
border-bottom: 4px solid rgba(46,82,101,200)</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;color: rgb(85, 255, 255);
background-color: rgba(0,0,0,0);
</string>
   </property>
   <property name="text">
    <string>USERNAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwd_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;color: rgb(85, 255, 255);
background-color: rgba(0,0,0,0);
</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="failure_info">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;
background-color: rgba(0,0,0,0);
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Arial&quot;;color: rgb(85, 255, 255);
background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>CREATE A FREE ACCOUNT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home_page">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
